features/symbol.yml

```diff
@@ -23,6 +23,5 @@ compat_features:
   - javascript.builtins.Symbol.toString
   - javascript.builtins.Symbol.toStringTag
   - javascript.builtins.Symbol.toStringTag.dom_objects
-  - javascript.builtins.Symbol.unscopables
```

Comment:
This key isn't _itself_ deprecated, but it exists to use with a deprecated feature. I've moved it into the deprecated feature.